controller/control.go

```diff
@@ -234,6 +234,11 @@ func (c *Controller) Start(addresses ...string) error {
 
 func (c *Controller) WriteAt(b []byte, off int64) (int, error) {
 	c.RLock()
+	if off < 0 || off+int64(len(b)) > c.size {
```

Comment:
Because the command I provided above is running in the host, rather inside container.